app/views/idv/in_person/address/show.html.erb

```diff
@@ -0,0 +1,95 @@
+<% content_for(:pre_flash_content) do %>
+  <%= render StepIndicatorComponent.new(
+        steps: Idv::Flows::InPersonFlow::STEP_INDICATOR_STEPS,
+        current_step: :verify_info,
+        locale_scope: 'idv',
+        class: 'margin-x-neg-2 margin-top-neg-4 tablet:margin-x-neg-6 tablet:margin-top-neg-4',
+      ) %>
+<% end %>
+
+<% title t('titles.doc_auth.verify') %>
```

Comment:
I have not tested  to confirm but wondering if this should be  <% title t('titles.doc_auth.address') %>
If I am able to test- I will let you know results

Reply:
i copied this from the other html.erb file and now am wondering why it might be in the file. This string reads "Verify your identity" but I don't see any text that says that on the page that is rendered. Hmmm, curious.

Reply:
Resolving comment because you removed it <% title t('titles.doc_auth.verify') %>.